constants.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+export description="Python client drivers for JanusGraph"
```

Comment:
Constants are typically written in all-uppercase, unless there's a strong need for them to be lowercase as in here (lowercase is typically used for mutable, local vars).

Reply:
Quick question @mbrukman : Since these constants are READ ONLY, can we declare them as `declare -r` like you mentioned bellow then do `source constants.sh` and then can we use them in Python like we do currently in `build.py` like `os.environ.get("PROJECT_NAME")` ?